src/screens/NationsScreen/NationsListScreen/index.js

```diff
@@ -58,7 +58,7 @@ class NationsListScreen extends Component {
             const nation = item.item;
             return (<NationListItem text={nation.nationName}
                                     onPress={this.props.onSelectItem}
-                                    status={i18n.t(`enums.nation.status.${resolveStatus(nation)}`)}
+                                    status={(resolveStatus(nation) === null ? '' : i18n.t(`enums.nation.status.${resolveStatus(nation).key}`))}
```

Comment:
Is it better to call resolveStatus once at the top and assign result to a const?